dev/breeze/src/airflow_breeze/commands/release_command.py

```diff
@@ -0,0 +1,284 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import os
+
+import click
+
+from airflow_breeze.utils.common_options import option_answer
+from airflow_breeze.utils.confirm import confirm_action
+from airflow_breeze.utils.console import console_print
+from airflow_breeze.utils.path_utils import AIRFLOW_SOURCES_ROOT
+from airflow_breeze.utils.run_utils import run_command
+
+CI = os.environ.get("CI")
+
+
+def clone_asf_repo(working_dir):
+    if confirm_action("Clone ASF repo?"):
+        run_command(["rm", "-rf", f"{working_dir}/asf-dist"])
+        run_command(
+            ["svn", "checkout", "--depth=immediates", "https://dist.apache.org/repos/dist", "asf-dist"]
+        )
+        dev_dir = f"{working_dir}/asf-dist/dev/airflow"
+        release_dir = f"{working_dir}/asf-dist/release/airflow"
+        run_command(["svn", "update", "--set-depth", "infinity", dev_dir])
+        run_command(["svn", "update", "--set-depth", "infinity", release_dir])
+
+
+def create_version_dir(version):
+    if CI:
```

Comment:
This?

Reply:
Yep. this we should skip. SVN is centralized system and it might fail without credentials

Reply:
Might be worth trying at least. I thought it didn't persist anything remote until you did the commit.